common/lib/xmodule/xmodule/video_module/transcripts_utils.py

```diff
@@ -640,25 +670,39 @@ def available_translations(self, transcripts, verify_assets=None):
 
         # If we've gotten this far, we're going to verify that the transcripts
         # being referenced are actually in the contentstore.
```

Comment:
We need to update/move the comment because right below the comment we are verifying val transcripts but comment is talking about content store.